test/integration-test.py

```diff
@@ -202,6 +200,36 @@ def run_node_test():
     wait_for_ocsp_revoked(cert_file_pem, "../test-ca.pem", ee_ocsp_url)
     return 0
 
+def run_caa_node_test():
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        s.connect(('localhost', 4000))
```

Comment:
This check is now implemented in startservers.py. I think it should be removed here, and above.